### DISCUSSION TITLE
Can jira-cli be used to re-rank issues in Todo?

### DISCUSSION THREAD

**Discussion by h0adp0re** [ankitpokhrel/jira-cli#592] [Ideas]
I need to automate adding issues to the top of Todo, is this possible with the Jira API?

---

Edit: I realized I can check the schema from the config file and there was indeed a "Rank" field. Alas, attempting to assign a value to the field results in this:
```
Error:
  - customfield_10009: Field 'customfield_10009' cannot be set. It is not on the appropriate screen, or unknown.

jira: Received unexpected response '400 '.
Please check the parameters you supplied and try again.
```

The command issued was this:
```sh
jira issue edit CODE-1234 --no-input --custom rank="Top"
```

**Comment by h0adp0re** [comment]
Bumping the question since it seems it's been neglected.

**Comment by ankitpokhrel** [comment]
Hi @h0adp0re, sorry I missed this one. The rank is a custom field in your installation, so I assume one of the two things might be happening.

1. Either the `Rank` field is not available when editing the issue (hence the error). Is this field available when editing the issue in the UI?
2. Or, the datatype of `Rank` field is something that is not supported by the tool yet. Can you check what datatype is listed for the field in config? More on that [here](https://github.com/ankitpokhrel/jira-cli/discussions/346).
